site/public/js/userform.js

```diff
@@ -204,9 +204,11 @@ function autoCompleteOnUserId(user_information) {
 }
 
 function completeUserFormInformation(user) {
-    var form = $("#edit-user-form");
+  var form = $("#edit-user-form");
```

Comment:
It looks like you removed some white-space here and that messes with the indentation. I think it looks better with the original white-space where everything lines up.
This also will require fixing the new line 209 to be at the same indentation level.